src/PowerShellEditorServices/Session/PSReadLinePromptContext.cs

```diff
@@ -30,7 +30,11 @@ internal class PSReadLinePromptContext : IPromptContext {
             [System.Diagnostics.DebuggerStepThrough()]
             param()
             end {
-                $module = Get-Module -ListAvailable PSReadLine | Where-Object Version -ge '2.0.0' | Sort-Object -Descending Version | Select-Object -First 1
+                $module = Get-Module -ListAvailable PSReadLine `
```

Comment:
Better to use the <kbd>|</kbd> character for line continuation here.  I like to format it like this:

```powershell
$module = Get-Module -ListAvailable PSReadLine |
    Where-Object Version -ge '2.0.0' |
    Where-Object {
        $psd = Import-PowerShellDataFile $_.Path
        $release = $psd.PrivateData.PSData.Prerelease
        $release -notin (@('beta1', 'beta2'))
    } |
    Sort-Object -Descending Version |
    Select-Object -First 1

Reply:
But my aligned pipes! Ah well, I'll take one for the team

Reply:
Hah yeah I hear ya.  I don't mind back ticks personally but the pitchforks come out **fast** for that 😄

Reply:
Agreed, especially when you're coding in VSCode with this setting - `"files.trimTrailingWhitespace": true,`.